<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_07">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_08">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_09">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_04">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_05">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_06">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_01">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_02">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_03">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_00">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_14">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_15">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_15">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_mult">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_16">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_16">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_div">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_17">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_17">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_allclear">
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_18">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>91</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_18">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>‚Üê</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_19">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_19">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_20">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_20">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>EXIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_operator">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_21">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_21">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_point">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_first">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_second">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_result">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_equal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mini Calculator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
